x-pack/plugin/downsample/src/main/java/org/elasticsearch/xpack/downsample/DownsampleMetrics.java

```diff
@@ -31,6 +31,9 @@ public class DownsampleMetrics extends AbstractLifecycleComponent {
 
     public static final String LATENCY_SHARD = "es.tsdb.downsample.latency.shard.histogram";
     public static final String LATENCY_TOTAL = "es.tsdb.downsample.latency.total.histogram";
+    public static final String SUCCESS = "es.tsdb.downsample.success.total";
+    public static final String FAILURE = "es.tsdb.downsample.failure.total";
+    public static final String INVALID_CONFIGURATION = "es.tsdb.downsample.invalid_configuration.total";
```

Comment:
I'd have a sinlge counter "es.tsdb.downsample.actions.total.counter" and "es.tsdb.downsample.actions.shard.counter" and use the dimension to break them down by status. Thoughts?

Reply:
I think these new metrics are are total metrics, right? (in other words not per shard)

Reply:
Indeed, I think we need metrics for both total and per-shard actions.